packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -381,24 +380,23 @@ export const VideoForm: FC<VideoFormProps> = memo(({ onSubmit, setFormStatus })
             validate: (value) => value !== null,
           }}
           render={({ field: { value, onChange } }) => (
-            <RadioCardButtonsContainer>
-              <OptionCardRadio
-                value="true"
-                label="Public"
-                onChange={() => onChange(true)}
-                selectedValue={value?.toString()}
-                helperText="Visible to all"
-                disabled={videoFieldsLocked}
-              />
-              <OptionCardRadio
-                value="false"
-                label="Unlisted"
-                onChange={() => onChange(false)}
-                selectedValue={value?.toString()}
-                helperText="Visible with link only"
-                disabled={videoFieldsLocked}
-              />
-            </RadioCardButtonsContainer>
+            <OptionCardGroupRadio
```

Comment:
Not working for me. I tried to edit my video and set it to unlisted and nothing happened once the transaction was done.